resources/shuttle-commands.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Shuttle Commands"
+---
+
+Before using the shuttle services it is important to get to know its different commands.
+To access the different shuttle commands you have to type in the following command in the directory where shuttle was installed:
+```
+cargo shuttle help
+```
+
+## Commands
+After that a list of different commands would appear we will be explaining each one separately in the following table.
```

Comment:
I forgot to tell you about this, but there is an additional trick here to get more info about the commands. I think we can replace this line with: "> Tip: Run a command with the `--help` flag to see available subcommands"